rest-server/src/controllers/job.js

```diff
@@ -114,5 +114,25 @@ const remove = (req, res) => {
   });
 };
 
+/**
+ * Start or stop job.
+ */
+const execute = (req, res) => {
+  req.body.username = req.user.username;
+  Job.prototype.stopJob(req.job.name, req.body, (err) => {
+    if (err) {
+      logger.warn('execute job %s error\n%s', req.job.name, err.stack);
+      return res.status(500).json({
+        error: 'JobExecuteError',
+        message: 'job execute error',
```

Comment:
>'job execute error [](start = 17, length = 18)

Can we set the message to include err.message here?

Reply:
err.message is set in `model/job.js`, print error stack then